spec/models/quiz_spec.rb

```diff
@@ -5,4 +5,19 @@
 
   it { should validate_presence_of(:name) }
   it { should validate_uniqueness_of(:name) }
+
+  it "returns score for student" do
+    user = FactoryGirl.create(:user_with_answered_quiz)
+    quiz = Quiz.first
+
+    expect(quiz.student_score(user)).to eq(1)
+  end
+
+  it "returns student's completion date of quiz" do
+    user = FactoryGirl.create(:user_with_answered_quiz)
+    time = Time.now.round
+    quiz = Quiz.first
```

Comment:
I totally agree for the first test ("returns score"), but for the second one ("completion date") is it okay if I leave it as is? My thought is that since it's time dependent and not actually answer-dependent, it seems kind of irrelevant to the reader to see what the user's answers actually are. 

(Also, I wasn't totally sure about the best way to run that "completion date" quiz, so I ended up with something that might not be ideal. Let me know if you want me to change how I'm testing the times!)